patches/ui-webui-resources-cr_elements-cr_icon-cr_icon.ts.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/ui/webui/resources/cr_elements/cr_icon/cr_icon.ts b/ui/webui/resources/cr_elements/cr_icon/cr_icon.ts
+index a71f644e0f03ea977bc609791209bf44619ff1d7..3adde5c39a992825db077193c27545b9e522e5b4 100644
+--- a/ui/webui/resources/cr_elements/cr_icon/cr_icon.ts
++++ b/ui/webui/resources/cr_elements/cr_icon/cr_icon.ts
+@@ -45,7 +45,7 @@ export class CrIconElement extends CrLitElement {
+     }
+   }
+ 
+-  private updateIcon_() {
++  /*private*/ updateIcon_() {
```

Comment:
@fallaciousreasoning Pls, let me know if there's a better way around this. We replace this function in `browser/resources/settings/brave_overrides/cr_icon.ts`

Reply:
Ah yup, definitely - `private` doesn't actually do anything at runtime so if you do

```ts
(CrIconElement.prototype as any).updateIcon_ = ...
```

then everything should still work :smile:

Reply:
That worked great. Thanks @fallaciousreasoning 👍